### PR TITLE
models in project

### PR DESCRIPTION
**Pull request by lmsamples**
## Proposed changes
Models folder should be added
## Steps To Test
_Please include expected output._
make sure models are all there and have correct data
## What kind of change does this PR introduce?
Please check one of the following:
- [ ] Bug fix
- [ ] Documentation update
- [x] New feature
- [ ] Other

### CODE REVIEW

File: BangazonAPI/Models/Computer.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+	public class Computer
+	{
```
Comment (jordan-castelloe) [inline BangazonAPI/Models/Computer.cs:9]: Hi! I think maybe there's supposed to be some properties in here-- are we missing some code?

File: BangazonAPI/Models/Department.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+	public class Class
+	{
```
Comment (jordan-castelloe) [inline BangazonAPI/Models/Department.cs:9]: Let's make sure that we name our classes

**Review by jordan-castelloe — CHANGES_REQUESTED** [review]
I think we're missing some code-- did something not get saved?

File: BangazonAPI/Models/Department.cs (hunk 2 above)
Comment (lmsamples) [reply BangazonAPI/Models/Department.cs:9]: should be fixed now

File: BangazonAPI/Models/Computer.cs (hunk 1 above)
Comment (lmsamples) [reply BangazonAPI/Models/Computer.cs:9]: yep i added it all back in there  check now

**Review by jordan-castelloe — APPROVED** [review]
Looks good! 👍

**Comment by calebooski13** [comment]
All the models look great and have the proper information in each

**Pull request merged**